test/e2e/cypress/integration/search.spec.js

```diff
@@ -436,7 +436,7 @@ describe('Search', function() {
       .and('contain', 'bar')
   })
 
-  it('should show aggregations in search result when aggregations are specified in the raw filter', function() {
+  it.skip('should show aggregations in search result when aggregations are specified in the raw filter', function() {
```

Comment:
Debug leftover?

Reply:
no, I have temporarily taken out the `aggregations` field from the `DocumentListItem` because it just didn't make sense: we showed _all the aggregations_ that resulted from the query in each item. We actually need a separate list (or a separate card that renders bare JSON) to display  the aggregations :point_right: #653